### PR TITLE
#389: Add HTML format

### LINKED ISSUE #389: Support HTML formatter for pretty HTML report
**Tailor Version:** v0.1.0

**Swift Version:** 1.0

**Platform (Mac/Linux/Windows/CI):** Mac

**Installation Method:** Homebrew

### Steps to Reproduce Issue
tailor --format=html

### PR DESCRIPTION
**Pull request by alykhank**
Resolves #389.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -263,6 +263,7 @@ dependencies {
     compile 'org.fusesource.jansi:jansi:1.11'
     compile 'org.yaml:snakeyaml:1.17'
     compile 'com.google.code.gson:gson:2.6.2'
+    compile 'com.github.spullara.mustache.java:compiler:0.9.1'
```
Comment (adityatrivedi) [inline build.gradle]: Licence needs to be added to the README.  It also needs to be added to the distribution.

Comment (alykhank) [inline build.gradle]: Done.

**Comment by nehasingh2293** [comment]
👍

**Comment by adityatrivedi** [comment]
👍

**Pull request merged**